go.mod

```diff
@@ -239,10 +238,12 @@ require (
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	github.com/power-devops/perfstat v0.0.0-20210106213030-5aafc221ea8c // indirect
 	github.com/prometheus/client_golang v1.15.1 // indirect
+	github.com/prometheus/common v0.42.0 // indirect
```

Comment:
why are we moving from `v0.43.0` to `v0.42.0 indirect`?

Reply:
Reverting it.